### ISSUE TITLE
Fine-tuning LingBot-VA with only 50 official episodes per task — is it sufficient?

### ISSUE THREAD

**Issue by SizheZhao** [Robbyant/lingbot-va#76]
Thanks for the great work. 

I am fine-tuning LingBot-VA on **[RMBench,](https://github.com/RoboTwin-Platform/RMBench) a benchmark very similar to RoboTwin**. Each task provides **50 official episodes**, which is exactly **the same data scale as RoboTwin 2.0**. So far, I have only used one task (battery_try) for a preliminary test:

- Training on A800 with 8 GPUs × 4 gradient accumulation = batch size 32
- After segmentation, there are 4 subtasks, each with about 50 segments
- Wan-VAE encodes 1 + 4*k frames
- The original 30 FPS data is downsampled to 15 FPS, and encoding is performed every 4 frames (except first frame)
- During inference, each chunk contains 2 frames, corresponding to 2 × 8 actions
- Norm:  q01/q99 for position, [0, 1] for quaternion

Below are the training losses. I used the 12K checkpoint for inference. The video generation (I2VA mode) quality is perfect, but the **robotic arm exhibits severe shaking during execution** (Simulation Evaluation).

My current guess is that training for **too many steps on only 50 episodes** may have caused overfitting. I noticed that LingBot-VA uses data augmentation on RoboTwin 2.0 (50 official episodes + 500 augmented episodes per task). I was wondering if you have tried fine-tuning LingBot-VA using only the 50 official episodes from RoboTwin 2.0?

<img width="1978" height="483" alt="Image" src="https://github.com/user-attachments/assets/cb3dfcc8-8127-48fc-8eb7-707622d0f42a" />

https://github.com/user-attachments/assets/70423fbe-903b-4a3d-a0db-efac70c66d08

**Comment by zqh0253** [comment]
Thanks for your interest. I would suggest splitting the dataset into a training and validation set, then evaluating the IDM results on both to check whether overfitting is occurring.

To assess IDM, you can kv-cache the ground-truth video latents and estimate actions during inference time.

**Comment by zqh0253** [comment]
Fig. 8 in our preprint shows few-shot performance.

**Comment by SizheZhao** [comment]
> Fig. 8 in our preprint shows few-shot performance.

Much appreciated!
